build.gradle

```diff
@@ -1,98 +1,76 @@
-
-  gradleLint.rules = ['all-dependency', 'dependency-parentheses', 'duplicate-dependency-class']
-
-  dependencies {
-    implementation 'org.slf4j:slf4j-api:1.7.26'
-    runtime 'org.slf4j:slf4j-simple:1.6.4'
-  }
-}
-
+/*
 subprojects {
   task sourcesJar(type: Jar) {
         from sourceSets.main.java
         archiveClassifier = 'sources'
     }
-}
+} */
 
-project(':common') {
-  dependencies {
-      implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
-      implementation 'org.jasypt:jasypt:1.9.2'
-  }
+repositories {
+  jcenter()
 }
 
-project(':configurator') {
-
-  mainClassName = 'org.apereo.openequella.adminconsole.configurator.Configurator'
+group = 'org.apereo.openequella.adminconsole'
+version = artifactVersion
+mainClassName = 'org.apereo.openequella.adminconsole.launcher.ClientLauncher'
 
-  apply plugin: "application"
-  
-  dependencies {
-    compile project(':common')
+apply plugin: 'java'
+apply plugin: 'base'
+apply plugin: 'nebula.lint'
+apply plugin: "application"
 
-    testImplementation "junit:junit:$junitVersion"
-  }
+// Gradlelint is broken: https://github.com/nebula-plugins/gradle-lint-plugin/issues/203
+gradleLint.rules = [/*'all-dependency', */'dependency-parentheses', 'duplicate-dependency-class']
 
-  // FIXME: gradle complains if this isn't listed in both positions...
-  mainClassName = 'org.apereo.openequella.adminconsole.configurator.Configurator'
+dependencies {
+  implementation 'org.slf4j:slf4j-api:1.7.26'
+  implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8' 
+  implementation 'org.jasypt:jasypt:1.9.2'
+  runtimeOnly 'org.slf4j:slf4j-simple:1.6.4'
+  testImplementation "junit:junit:$junitVersion"
+}
 
-  jar {
-    manifest {
+jar {
+  //dependsOn 'clean', 'build'
+  archiveFileName = 'admin.jar'
+  manifest {
       attributes('Implementation-Title': project.name,
               'Implementation-Version': project.version,
               'Main-Class': mainClassName)
-    }
   }
 }
 
-project(':launcher') {
-
-  mainClassName = 'org.apereo.openequella.adminconsole.launcher.ClientLauncher'
+task cleanDist() {
+  dependsOn 'clean'
+  doLast {
+    delete 'dist'
+  }
+}
 
-  apply plugin: "application"
+task copyDistArtifacts(type: Copy) {
+  dependsOn 'copyScripts'
 
-  group = 'org.apereo.openequella.adminconsole'
+  from(file('build/libs/admin.jar'))
+  from configurations.default
+  into('dist/lib')
+}
 
-  dependencies {
-    compile project(':common')
+task copyScripts(type: Copy){
+  dependsOn 'cleanDist', 'jar'
 
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
-     
-    testImplementation "junit:junit:$junitVersion"
-  }
+  from(file('scripts'))
+  into('dist')
+  include('**/*')
+}
 
-  // FIXME: gradle complains if this isn't listed in both positions...
-  mainClassName = 'org.apereo.openequella.adminconsole.launcher.ClientLauncher'
+task dist {
+  description 'Copies everything required for the distributable into a dist folder'
```

Comment:
This task needs to be triggered by another task such as build

Reply:
No, it was meant to be a new task that you can run.  It couple just be included in build though I guess.